ext/testing/mod.js

```diff
@@ -429,27 +372,32 @@ class TestContext {
 		const testContext = new TestContext(name, parent);
 
 		parent?.lock(testContext);
+		op_set_promise_sanitized_test_name(name);
+		op_set_promise_sanitization_hook();
 		const response = callback(testContext);
-
 		if (response instanceof Promise) {
 			return response.then(() => {
+				// We allow two pending promises for the test itself
+				// And second for awaiting the test
+				// In case this assumption is wrong, program will still throw at the end of the event loop
+				TestContext.sanitizeAsyncOps(testContext, 2);
```

Comment:
Will it throw with the correct test name?

Reply:
Yes, because that promise has to be `await`ed